src/controllers/controller.bar.js

```diff
@@ -227,7 +239,7 @@ module.exports = function(Chart) {
 			rectangle._model = {
 				datasetLabel: dataset.label,
 				label: chart.data.labels[index],
-				borderSkipped: custom.borderSkipped ? custom.borderSkipped : rectangleOptions.borderSkipped,
+				borderSkipped: custom.borderSkipped || custom.borderSkipped === null ? custom.borderSkipped : rectangleOptions.borderSkipped,
```

Comment:
I don't understand what you're checking for here. Won't `custom.borderSkipped || custom.borderSkipped === null` almost always be true? Are you trying to do `custom.borderSkipped !== undefined`? We rarely would check for `custom.borderSkipped === null` and instead would usually do `helpers.isNullOrUndef(custom.borderSkipped)`